functionary/ui/forms/tasks.py

```diff
@@ -122,7 +111,7 @@ def __init__(self, function, data=None, initial=None, prefix="task-parameter"):
             kwargs = {
                 "label": value["title"],
                 "label_suffix": param_type,
-                "initial": _prepare_initial_value(param_type, initial_value),
```

Comment:
I know there was some back and forth, but we landed on this needing to stay.  The thing that needs to change is that `_transform_initial_mapping` should no longer just point to `json.loads` as the transformation function, but instead some other helper that calls `json.loads` only if necessary (the value is actually a string).

As it is now, if someone provides a default value for a json parameter, the value gets populated in the form as an escaped string, like:

`"{\"param1\": \"value1\", \"param2\": \"value2\"}"`

The whole `_preapre_initial_value` thing is there to make sure it instead is populated as:

`{"param1": "value1", "param2": "value2"}`

Reply:
Implemented the helper function